modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb

```diff
@@ -86,7 +86,7 @@ module EBenefitsBenefitClaimStatusWebServiceBean
           Bean.new(
             path: 'EBenefitsBnftClaimStatusWebServiceBean',
             namespaces: Namespaces.new(
-              target: 'http://services.share.benefits.vba.va.gov/',
+              target: 'http://claimstatus.services.ebenefits.vba.va.gov/',
```

Comment:
interesting, why the change here? was the former not actually a namespace for this service?

Reply:
Yeah, I had that entered in incorrectly originally.